### PR TITLE
Updates to clarify that automatic version tagging only works for APM …

### PR DESCRIPTION
**Pull request by lee-brian**
…data

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 1]
```diff
@@ -107,14 +110,14 @@ metadata:
   labels:
     tags.datadoghq.com/env: "<ENV>"
     tags.datadoghq.com/service: "<SERVICE>"
-    # tags.datadoghq.com/version: "<VERSION>" #Uncomment this line to manually tag your version
+    tags.datadoghq.com/version: "<VERSION>" #Comment out this line to use automatic version tagging in APM
```
Comment (jdgumz) [inline content/en/getting_started/tagging/unified_service_tagging.md:113]: Commenting out this line will result in no version being set for logs/metrics though.

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 2]
```diff
@@ -126,10 +129,10 @@ template:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.labels['tags.datadoghq.com/service']
-          # - name: DD_VERSION #Uncomment this line to manually tag your version
-          #   valueFrom: #Uncomment this line to manually tag your version
-          #     fieldRef: #Uncomment this line to manually tag your version
-          #       fieldPath: metadata.labels['tags.datadoghq.com/version'] #Uncomment this line to manually tag your version
+          - name: DD_VERSION #Comment out this line to use automatic version tagging in APM
```
Comment (jdgumz) [inline content/en/getting_started/tagging/unified_service_tagging.md:132]: I think this is the only thing that you'd want to comment out.

From the labels alone we should be able to put version on logs/metrics.

It's technically only the APM tracer that needs the `DD_VERSION` env var in order to set version itself.

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 1 above)
Comment (lee-brian) [reply content/en/getting_started/tagging/unified_service_tagging.md]: I removed messaging to comment this out

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 2 above)
Comment (lee-brian) [reply content/en/getting_started/tagging/unified_service_tagging.md]: Removed all messaging to comment this out, to reduce confusion until we have this feature enabled for all product data

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 3]
```diff
@@ -272,24 +259,18 @@ If your service has no need for the Datadog environment variables (for example,
...
-
-[1]: /tracing/services/deployment_tracking/
-[2]: /watchdog/faulty_deployment_detection/
-[3]: /integrations/guide/source-code-integration/?tab=go#embed-git-information-in-your-build-artifacts
 
-{{% /tab %}}
+##### Automatic version tagging for APM data in containerized environments
 
-{{% tab "ECS" %}}
+<div class="alert alert-info">This feature is only enabled for <a href="https://docs.datadoghq.com/tracing/">Application Performance Monitoring (APM)</a> data.</div>
```
Comment (brett0000FF) [inline content/en/getting_started/tagging/unified_service_tagging.md:269]:
```suggestion
<div class="alert alert-info">This feature is only enabled for <a href="/tracing/">Application Performance Monitoring (APM)</a> data.</div>
```

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 4]
```diff
@@ -341,14 +325,30 @@ If your service has no need for the Datadog environment variables (for example,
...
-   * ,"com.datadoghq.tags.version": "<VERSION>"
-   **/
+  "com.datadoghq.tags.service": "<SERVICE>",
+  "com.datadoghq.tags.version": "<VERSION>"
 }
 ```
 
+##### Automatic version tagging for APM data in containerized environments
+
+<div class="alert alert-info">This feature is only enabled for <a href="https://docs.datadoghq.com/tracing/">Application Performance Monitoring (APM)</a> data.</div>
```
Comment (brett0000FF) [inline content/en/getting_started/tagging/unified_service_tagging.md:335]:
```suggestion
<div class="alert alert-info">This feature is only enabled for <a href="/tracing/">Application Performance Monitoring (APM)</a> data.</div>
```

**Review by brett0000FF — APPROVED** [review]
Thanks, this looks good! I left two super minor suggestions that are not wrong, just a best practice.

**Pull request merged**